template-compose/app/build.gradle.kts

```
@@ -133,54 +134,25 @@ dependencies {
 
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
-    with(Dependencies.AndroidX) {
-        implementation(CORE_KTX)
-        implementation(LIFECYCLE_RUNTIME_KTX)
-        implementation(LIFECYCLE_RUNTIME_COMPOSE)
-        implementation(DATASTORE_PREFERENCES)
-    }
-
-    with(Dependencies.Compose) {
-        implementation(platform(BOM))
-        implementation(UI)
-        debugImplementation(UI_TOOLING)
-        implementation(UI_TOOLING_PREVIEW)
-        implementation(MATERIAL)
-        implementation(NAVIGATION)
-
-        implementation(ACCOMPANIST_PERMISSIONS)
-    }
+    implementation(libs.bundles.androidx)
 
-    with(Dependencies.Hilt) {
-        implementation(ANDROID)
-        implementation(NAVIGATION_COMPOSE)
-        kapt(COMPILER)
-    }
+    implementation(platform(libs.compose.bom))
+    implementation(libs.bundles.compose)
+    debugImplementation(libs.compose.ui.tooling)
 
-    with(Dependencies.Log) {
-        implementation(TIMBER)
+    implementation(libs.bundles.hilt)
+    kapt(libs.hilt.compiler)
 
-        debugImplementation(CHUCKER)
-        releaseImplementation(CHUCKER_NO_OP)
-    }
+    implementation(libs.timber)
+    debugImplementation(libs.chucker)
+    releaseImplementation(libs.chucker.no.op)
 
-    with(Dependencies.Util) {
-        implementation(COMMON_KTX)
-    }
+    implementation(libs.nimble.common)
 
-    with(Dependencies.Test) {
-        // Unit test
-        testImplementation(COROUTINES)
-        testImplementation(JUNIT)
-        testImplementation(KOTEST)
-        testImplementation(MOCKK)
-        testImplementation(TURBINE)
-
-        // UI test with Robolectric
-        testImplementation(platform(Dependencies.Compose.BOM))
-        testImplementation(COMPOSE_UI_TEST_JUNIT)
-        testImplementation(ROBOLECTRIC)
-    }
+    testImplementation(libs.bundles.unitTest)
+    testImplementation(libs.test.turbine)
+    testImplementation(platform(libs.compose.bom))
+    testImplementation(libs.bundles.implementationTest)
```

I think we can still keep the comments 

```suggestion
    // Unit test
    testImplementation(libs.bundles.unitTest)
    testImplementation(libs.test.turbine)

    // UI test with Robolectric
    testImplementation(platform(libs.compose.bom))
    testImplementation(libs.bundles.implementationTest)
```